generator/Makefile

```diff
@@ -184,7 +184,8 @@ $(MIBDIR)/.dell:
 	@curl $(CURL_OPTS) $(CURL_USER_AGENT) -o $(TMP) $(DELL_URL)
 	@unzip -j -d $(MIBDIR) $(TMP) support/station/mibs/iDRAC-*.mib
 	# There are some additional characters behind MIB end that break parsing
-	@sed -i '$ d' $(MIBDIR)/iDRAC-SMIv1.mib
+	@sed '$ d' $(MIBDIR)/iDRAC-SMIv1.mib > $(MIBDIR)/iDRAC-SMIv1.mib.temp
```

Comment:
Take a look at the `apc-powernet-mib` sed, using `-i.bak` is portable between GNU and BSD sed.